client/html/pages/panel/settings/index.html

```diff
@@ -25,7 +25,7 @@
 
                 <!-- BEGIN CONTACTS NAV -->
                 <div class="col-sm-5 col-md-4 col-lg-3">
-                  <ul class="nav nav-pills nav-stacked nav-tab" data-toggle="tabs">
+                  <ul class="nav nav-pills nav-stacked nav-tabs-vert" data-toggle="tabs">
```

Comment:
Why do we need this class **nav-tabs-vert**? is it for css rules? if so, leave both **nav-tabs** and **nav-tabs-vert** (if possible) then we wont't need to create one more event handler